LoopKitUI/Views/GuideNavigationButton.swift

```diff
@@ -41,7 +41,7 @@ public struct GuideNavigationButton<Destination>: View where Destination: View {
                 self.navigationLinkIsActive.toggle()
             }) {
                 Text(label)
-                    .guideButtonStyle(buttonStyle)
+                    .actionButtonStyle(buttonStyle)
```

Comment:
I see this isn't in sync with `LoopKit` `dev`.

Reply:
Updated